Odoo Script Installation

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Odoo

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/odoo.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📈 Which Proxmox version are you on?

9.0.10

### 📝 Provide a clear and concise description of the issue.

Installing script stops midway


### 🔄 Steps to reproduce the issue.

Run the command with default or advanced settings.

### ❌ Paste the full error output (if available).

   ____      __          
  / __ \____/ /___  ____ 
 / / / / __  / __ \/ __ \
/ /_/ / /_/ / /_/ / /_/ /
\____/\__,_/\____/\____/ 
                         
  ⚙️  Using Default Settings on node pvi (🔍  Verbose)
  🆔  Container ID: 124
  🖥️  Operating System: debian (12)
  📦  Container Type: Unprivileged
  💾  Disk Size: 6 GB
  🧠  CPU Cores: 2
  🛠️  RAM Size: 2048 MiB
  🔍  Verbose Mode: Enabled
  🚀  Creating a Odoo LXC using the above default settings
  
  ✔️   Storage cephfs (Free: 5.2TB  Used: 435.0GB) [Template]
  ✔️   Storage local-lvm (Free: 242.5GB  Used: 98.3GB) [Container]
  ✔️   Cluster is quorate
  ✔️   Template debian-12-standard_12.12-1_amd64.tar.zst [online]
   💡   Template debian-12-standard_12.12-1_amd64.tar.zst is missing or corrupted. Re-downloading.
  ✔️   Template download successful.
  ✔️   LXC Container 124 was successfully created.
  ✔️   Started LXC Container
  ✔️   Network in LXC is reachable (ping)
  ✔️   Customized LXC Container
  ✔️   Set up Container OS
  ✔️   Network Connected: 192.168.0.168 
  ✔️   IPv4 Internet Connected
   ✖️   IPv6 Internet Not Connected
  ✔️   Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
Hit:1 http://deb.debian.org/debian bookworm InRelease
Hit:2 http://security.debian.org bookworm-security InRelease
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
  ✔️   Updated Container OS
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  binutils binutils-common binutils-x86-64-linux-gnu cpp cpp-12 dpkg-dev fakeroot fontconfig-config fonts-dejavu-core g++ g++-12 gcc gcc-12 libabsl20220623 libalgorithm-diff-perl libalgorithm-diff-xs-perl
  libalgorithm-merge-perl libaom3 libasan8 libatomic1 libavif15 libbinutils libc-dev-bin libc-devtools libc6-dev libcc1-0 libcrypt-dev libctf-nobfd0 libctf0 libdav1d6 libde265-0 libdeflate0 libdpkg-perl
  libfakeroot libfile-fcntllock-perl libfontconfig1 libfreetype6 libgav1-1 libgcc-12-dev libgd3 libgomp1 libgprofng0 libheif1 libisl23 libitm1 libjbig0 libjpeg62-turbo liblerc4 liblsan0 libmpc3 libmpfr6
  libnsl-dev libnuma1 libpng16-16 libquadmath0 librav1e0 libstdc++-12-dev libsvtav1enc1 libtiff6 libtirpc-dev libtsan2 libubsan1 libwebp7 libx11-6 libx11-data libx265-199 libxau6 libxcb1 libxdmcp6 libxpm4
  libyuv0 linux-libc-dev manpages-dev patch rpcsvc-proto
Suggested packages:
  binutils-doc cpp-doc gcc-12-locales cpp-12-doc debian-keyring g++-multilib g++-12-multilib gcc-12-doc gcc-multilib autoconf automake libtool flex bison gdb gcc-doc gcc-12-multilib glibc-doc git bzr
  libgd-tools libstdc++-12-doc make-doc ed diffutils-doc
The following NEW packages will be installed:
  binutils binutils-common binutils-x86-64-linux-gnu build-essential cpp cpp-12 dpkg-dev fakeroot fontconfig-config fonts-dejavu-core g++ g++-12 gcc gcc-12 libabsl20220623 libalgorithm-diff-perl
  libalgorithm-diff-xs-perl libalgorithm-merge-perl libaom3 libasan8 libatomic1 libavif15 libbinutils libc-dev-bin libc-devtools libc6-dev libcc1-0 libcrypt-dev libctf-nobfd0 libctf0 libdav1d6 libde265-0
  libdeflate0 libdpkg-perl libfakeroot libfile-fcntllock-perl libfontconfig1 libfreetype6 libgav1-1 libgcc-12-dev libgd3 libgomp1 libgprofng0 libheif1 libisl23 libitm1 libjbig0 libjpeg62-turbo liblerc4
  liblsan0 libmpc3 libmpfr6 libnsl-dev libnuma1 libpng16-16 libquadmath0 librav1e0 libstdc++-12-dev libsvtav1enc1 libtiff6 libtirpc-dev libtsan2 libubsan1 libwebp7 libx11-6 libx11-data libx265-199 libxau6
  libxcb1 libxdmcp6 libxpm4 libyuv0 linux-libc-dev make manpages-dev patch rpcsvc-proto
0 upgraded, 77 newly installed, 0 to remove and 0 not upgraded.
Need to get 80.7 MB of archives.
After this operation, 320 MB of additional disk space will be used.
Get:1 http://security.debian.org bookworm-security/main amd64 linux-libc-dev amd64 6.1.153-1 [2,192 kB]
Get:2 http://deb.debian.org/debian bookworm/main amd64 binutils-common amd64 2.40-2 [2,487 kB]
Get:3 http://deb.debian.org/debian bookworm/main amd64 libbinutils amd64 2.40-2 [572 kB]
Get:4 http://deb.debian.org/debian bookworm/main amd64 libctf-nobfd0 amd64 2.40-2 [153 kB]
Get:5 http://deb.debian.org/debian bookworm/main amd64 libctf0 amd64 2.40-2 [89.8 kB]
Get:6 http://deb.debian.org/debian bookworm/main amd64 libgprofng0 amd64 2.40-2 [812 kB]
Get:7 http://deb.debian.org/debian bookworm/main amd64 binutils-x86-64-linux-gnu amd64 2.40-2 [2,246 kB]
Get:8 http://deb.debian.org/debian bookworm/main amd64 binutils amd64 2.40-2 [65.0 kB]
Get:9 http://deb.debian.org/debian bookworm/main amd64 libc-dev-bin amd64 2.36-9+deb12u13 [47.4 kB]
Get:10 http://deb.debian.org/debian bookworm/main amd64 libcrypt-dev amd64 1:4.4.33-2 [118 kB]
Get:11 http://deb.debian.org/debian bookworm/main amd64 libtirpc-dev amd64 1.3.3+ds-1 [191 kB]
Get:12 http://deb.debian.org/debian bookworm/main amd64 libnsl-dev amd64 1.3.0-2 [66.4 kB]
Get:13 http://deb.debian.org/debian bookworm/main amd64 rpcsvc-proto amd64 1.4.3-1 [63.3 kB]
Get:14 http://deb.debian.org/debian bookworm/main amd64 libc6-dev amd64 2.36-9+deb12u13 [1,904 kB]
Get:15 http://deb.debian.org/debian bookworm/main amd64 libisl23 amd64 0.25-1.1 [683 kB]
Get:16 http://deb.debian.org/debian bookworm/main amd64 libmpfr6 amd64 4.2.0-1 [701 kB]
Get:17 http://deb.debian.org/debian bookworm/main amd64 libmpc3 amd64 1.3.1-1 [51.5 kB]
Get:18 http://deb.debian.org/debian bookworm/main amd64 cpp-12 amd64 12.2.0-14+deb12u1 [9,768 kB]
Get:19 http://deb.debian.org/debian bookworm/main amd64 cpp amd64 4:12.2.0-3 [6,836 B]
Get:20 http://deb.debian.org/debian bookworm/main amd64 libcc1-0 amd64 12.2.0-14+deb12u1 [41.7 kB]
Get:21 http://deb.debian.org/debian bookworm/main amd64 libgomp1 amd64 12.2.0-14+deb12u1 [116 kB]
Get:22 http://deb.debian.org/debian bookworm/main amd64 libitm1 amd64 12.2.0-14+deb12u1 [26.1 kB]
Get:23 http://deb.debian.org/debian bookworm/main amd64 libatomic1 amd64 12.2.0-14+deb12u1 [9,376 B]
Get:24 http://deb.debian.org/debian bookworm/main amd64 libasan8 amd64 12.2.0-14+deb12u1 [2,193 kB]
Get:25 http://deb.debian.org/debian bookworm/main amd64 liblsan0 amd64 12.2.0-14+deb12u1 [969 kB]
Get:26 http://deb.debian.org/debian bookworm/main amd64 libtsan2 amd64 12.2.0-14+deb12u1 [2,197 kB]
Get:27 http://deb.debian.org/debian bookworm/main amd64 libubsan1 amd64 12.2.0-14+deb12u1 [883 kB]
Get:28 http://deb.debian.org/debian bookworm/main amd64 libquadmath0 amd64 12.2.0-14+deb12u1 [145 kB]
Get:29 http://deb.debian.org/debian bookworm/main amd64 libgcc-12-dev amd64 12.2.0-14+deb12u1 [2,437 kB]
Get:30 http://deb.debian.org/debian bookworm/main amd64 gcc-12 amd64 12.2.0-14+deb12u1 [19.3 MB]
Get:31 http://deb.debian.org/debian bookworm/main amd64 gcc amd64 4:12.2.0-3 [5,216 B]
Get:32 http://deb.debian.org/debian bookworm/main amd64 libstdc++-12-dev amd64 12.2.0-14+deb12u1 [2,047 kB]
Get:33 http://deb.debian.org/debian bookworm/main amd64 g++-12 amd64 12.2.0-14+deb12u1 [10.7 MB]
Get:34 http://deb.debian.org/debian bookworm/main amd64 g++ amd64 4:12.2.0-3 [1,356 B]
Get:35 http://deb.debian.org/debian bookworm/main amd64 make amd64 4.3-4.1 [396 kB]
Get:36 http://deb.debian.org/debian bookworm/main amd64 libdpkg-perl all 1.21.22 [603 kB]
Get:37 http://deb.debian.org/debian bookworm/main amd64 patch amd64 2.7.6-7 [128 kB]
Get:38 http://deb.debian.org/debian bookworm/main amd64 dpkg-dev all 1.21.22 [1,353 kB]
Get:39 http://deb.debian.org/debian bookworm/main amd64 build-essential amd64 12.9 [7,704 B]
Get:40 http://deb.debian.org/debian bookworm/main amd64 libfakeroot amd64 1.31-1.2 [28.3 kB]
Get:41 http://deb.debian.org/debian bookworm/main amd64 fakeroot amd64 1.31-1.2 [66.9 kB]
Get:42 http://deb.debian.org/debian bookworm/main amd64 fonts-dejavu-core all 2.37-6 [1,068 kB]
Get:43 http://deb.debian.org/debian bookworm/main amd64 fontconfig-config amd64 2.14.1-4 [315 kB]
Get:44 http://deb.debian.org/debian bookworm/main amd64 libabsl20220623 amd64 20220623.1-1+deb12u2 [391 kB]
Get:45 http://deb.debian.org/debian bookworm/main amd64 libalgorithm-diff-perl all 1.201-1 [43.3 kB]
Get:46 http://deb.debian.org/debian bookworm/main amd64 libalgorithm-diff-xs-perl amd64 0.04-8+b1 [11.4 kB]
Get:47 http://deb.debian.org/debian bookworm/main amd64 libalgorithm-merge-perl all 0.08-5 [11.8 kB]
Get:48 http://deb.debian.org/debian bookworm/main amd64 libaom3 amd64 3.6.0-1+deb12u2 [1,850 kB]
Get:49 http://deb.debian.org/debian bookworm/main amd64 libdav1d6 amd64 1.0.0-2+deb12u1 [513 kB]
Get:50 http://deb.debian.org/debian bookworm/main amd64 libgav1-1 amd64 0.18.0-1+b1 [332 kB]
Get:51 http://deb.debian.org/debian bookworm/main amd64 librav1e0 amd64 0.5.1-6 [763 kB]
Get:52 http://deb.debian.org/debian bookworm/main amd64 libsvtav1enc1 amd64 1.4.1+dfsg-1 [2,121 kB]
Get:53 http://deb.debian.org/debian bookworm/main amd64 libjpeg62-turbo amd64 1:2.1.5-2 [166 kB]
Get:54 http://deb.debian.org/debian bookworm/main amd64 libyuv0 amd64 0.0~git20230123.b2528b0-1 [168 kB]
Get:55 http://deb.debian.org/debian bookworm/main amd64 libavif15 amd64 0.11.1-1+deb12u1 [94.4 kB]
Get:56 http://deb.debian.org/debian bookworm/main amd64 libpng16-16 amd64 1.6.39-2 [276 kB]
Get:57 http://deb.debian.org/debian bookworm/main amd64 libfreetype6 amd64 2.12.1+dfsg-5+deb12u4 [398 kB]
Get:58 http://deb.debian.org/debian bookworm/main amd64 libfontconfig1 amd64 2.14.1-4 [386 kB]
Get:59 http://deb.debian.org/debian bookworm/main amd64 libde265-0 amd64 1.0.11-1+deb12u2 [185 kB]
Get:60 http://deb.debian.org/debian bookworm/main amd64 libnuma1 amd64 2.0.16-1 [21.0 kB]
Get:61 http://deb.debian.org/debian bookworm/main amd64 libx265-199 amd64 3.5-2+b1 [1,150 kB]
Get:62 http://deb.debian.org/debian bookworm/main amd64 libheif1 amd64 1.15.1-1+deb12u1 [215 kB]
Get:63 http://deb.debian.org/debian bookworm/main amd64 libdeflate0 amd64 1.14-1 [61.4 kB]
Get:64 http://deb.debian.org/debian bookworm/main amd64 libjbig0 amd64 2.1-6.1 [31.7 kB]
Get:65 http://deb.debian.org/debian bookworm/main amd64 liblerc4 amd64 4.0.0+ds-2 [170 kB]
Get:66 http://deb.debian.org/debian bookworm/main amd64 libwebp7 amd64 1.2.4-0.2+deb12u1 [286 kB]
Get:67 http://deb.debian.org/debian bookworm/main amd64 libtiff6 amd64 4.5.0-6+deb12u2 [316 kB]
Get:68 http://deb.debian.org/debian bookworm/main amd64 libxau6 amd64 1:1.0.9-1 [19.7 kB]
Get:69 http://deb.debian.org/debian bookworm/main amd64 libxdmcp6 amd64 1:1.1.2-3 [26.3 kB]
Get:70 http://deb.debian.org/debian bookworm/main amd64 libxcb1 amd64 1.15-1 [144 kB]
Get:71 http://deb.debian.org/debian bookworm/main amd64 libx11-data all 2:1.8.4-2+deb12u2 [292 kB]
Get:72 http://deb.debian.org/debian bookworm/main amd64 libx11-6 amd64 2:1.8.4-2+deb12u2 [760 kB]
Get:73 http://deb.debian.org/debian bookworm/main amd64 libxpm4 amd64 1:3.5.12-1.1+deb12u1 [48.6 kB]
Get:74 http://deb.debian.org/debian bookworm/main amd64 libgd3 amd64 2.3.3-9 [124 kB]
Get:75 http://deb.debian.org/debian bookworm/main amd64 libc-devtools amd64 2.36-9+deb12u13 [55.0 kB]
Get:76 http://deb.debian.org/debian bookworm/main amd64 libfile-fcntllock-perl amd64 0.22-4+b1 [34.8 kB]
Get:77 http://deb.debian.org/debian bookworm/main amd64 manpages-dev all 6.03-2 [2,030 kB]
Fetched 80.7 MB in 1s (61.1 MB/s)      
Extracting templates from packages: 100%
Preconfiguring packages ...
Selecting previously unselected package binutils-common:amd64.
(Reading database ... 20346 files and directories currently installed.)
Preparing to unpack .../00-binutils-common_2.40-2_amd64.deb ...
Unpacking binutils-common:amd64 (2.40-2) ...
Selecting previously unselected package libbinutils:amd64.
Preparing to unpack .../01-libbinutils_2.40-2_amd64.deb ...
Unpacking libbinutils:amd64 (2.40-2) ...
Selecting previously unselected package libctf-nobfd0:amd64.
Preparing to unpack .../02-libctf-nobfd0_2.40-2_amd64.deb ...
Unpacking libctf-nobfd0:amd64 (2.40-2) ...
Selecting previously unselected package libctf0:amd64.
Preparing to unpack .../03-libctf0_2.40-2_amd64.deb ...
Unpacking libctf0:amd64 (2.40-2) ...
Selecting previously unselected package libgprofng0:amd64.
Preparing to unpack .../04-libgprofng0_2.40-2_amd64.deb ...
Unpacking libgprofng0:amd64 (2.40-2) ...
Selecting previously unselected package binutils-x86-64-linux-gnu.
Preparing to unpack .../05-binutils-x86-64-linux-gnu_2.40-2_amd64.deb ...
Unpacking binutils-x86-64-linux-gnu (2.40-2) ...
Selecting previously unselected package binutils.
Preparing to unpack .../06-binutils_2.40-2_amd64.deb ...
Unpacking binutils (2.40-2) ...
Selecting previously unselected package libc-dev-bin.
Preparing to unpack .../07-libc-dev-bin_2.36-9+deb12u13_amd64.deb ...
Unpacking libc-dev-bin (2.36-9+deb12u13) ...
Selecting previously unselected package linux-libc-dev:amd64.
Preparing to unpack .../08-linux-libc-dev_6.1.153-1_amd64.deb ...
Unpacking linux-libc-dev:amd64 (6.1.153-1) ...
Selecting previously unselected package libcrypt-dev:amd64.
Preparing to unpack .../09-libcrypt-dev_1%3a4.4.33-2_amd64.deb ...
Unpacking libcrypt-dev:amd64 (1:4.4.33-2) ...
Selecting previously unselected package libtirpc-dev:amd64.
Preparing to unpack .../10-libtirpc-dev_1.3.3+ds-1_amd64.deb ...
Unpacking libtirpc-dev:amd64 (1.3.3+ds-1) ...
Selecting previously unselected package libnsl-dev:amd64.
Preparing to unpack .../11-libnsl-dev_1.3.0-2_amd64.deb ...
Unpacking libnsl-dev:amd64 (1.3.0-2) ...
Selecting previously unselected package rpcsvc-proto.
Preparing to unpack .../12-rpcsvc-proto_1.4.3-1_amd64.deb ...
Unpacking rpcsvc-proto (1.4.3-1) ...
Selecting previously unselected package libc6-dev:amd64.
Preparing to unpack .../13-libc6-dev_2.36-9+deb12u13_amd64.deb ...
Unpacking libc6-dev:amd64 (2.36-9+deb12u13) ...
Selecting previously unselected package libisl23:amd64.
Preparing to unpack .../14-libisl23_0.25-1.1_amd64.deb ...
Unpacking libisl23:amd64 (0.25-1.1) ...
Selecting previously unselected package libmpfr6:amd64.
Preparing to unpack .../15-libmpfr6_4.2.0-1_amd64.deb ...
Unpacking libmpfr6:amd64 (4.2.0-1) ...
Selecting previously unselected package libmpc3:amd64.
Preparing to unpack .../16-libmpc3_1.3.1-1_amd64.deb ...
Unpacking libmpc3:amd64 (1.3.1-1) ...
Selecting previously unselected package cpp-12.
Preparing to unpack .../17-cpp-12_12.2.0-14+deb12u1_amd64.deb ...
Unpacking cpp-12 (12.2.0-14+deb12u1) ...
Selecting previously unselected package cpp.
Preparing to unpack .../18-cpp_4%3a12.2.0-3_amd64.deb ...
Unpacking cpp (4:12.2.0-3) ...
Selecting previously unselected package libcc1-0:amd64.
Preparing to unpack .../19-libcc1-0_12.2.0-14+deb12u1_amd64.deb ...
Unpacking libcc1-0:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package libgomp1:amd64.
Preparing to unpack .../20-libgomp1_12.2.0-14+deb12u1_amd64.deb ...
Unpacking libgomp1:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package libitm1:amd64.
Preparing to unpack .../21-libitm1_12.2.0-14+deb12u1_amd64.deb ...
Unpacking libitm1:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package libatomic1:amd64.
Preparing to unpack .../22-libatomic1_12.2.0-14+deb12u1_amd64.deb ...
Unpacking libatomic1:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package libasan8:amd64.
Preparing to unpack .../23-libasan8_12.2.0-14+deb12u1_amd64.deb ...
Unpacking libasan8:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package liblsan0:amd64.
Preparing to unpack .../24-liblsan0_12.2.0-14+deb12u1_amd64.deb ...
Unpacking liblsan0:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package libtsan2:amd64.
Preparing to unpack .../25-libtsan2_12.2.0-14+deb12u1_amd64.deb ...
Unpacking libtsan2:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package libubsan1:amd64.
Preparing to unpack .../26-libubsan1_12.2.0-14+deb12u1_amd64.deb ...
Unpacking libubsan1:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package libquadmath0:amd64.
Preparing to unpack .../27-libquadmath0_12.2.0-14+deb12u1_amd64.deb ...
Unpacking libquadmath0:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package libgcc-12-dev:amd64.
Preparing to unpack .../28-libgcc-12-dev_12.2.0-14+deb12u1_amd64.deb ...
Unpacking libgcc-12-dev:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package gcc-12.
Preparing to unpack .../29-gcc-12_12.2.0-14+deb12u1_amd64.deb ...
Unpacking gcc-12 (12.2.0-14+deb12u1) ...
Selecting previously unselected package gcc.
Preparing to unpack .../30-gcc_4%3a12.2.0-3_amd64.deb ...
Unpacking gcc (4:12.2.0-3) ...
Selecting previously unselected package libstdc++-12-dev:amd64.
Preparing to unpack .../31-libstdc++-12-dev_12.2.0-14+deb12u1_amd64.deb ...
Unpacking libstdc++-12-dev:amd64 (12.2.0-14+deb12u1) ...
Selecting previously unselected package g++-12.
Preparing to unpack .../32-g++-12_12.2.0-14+deb12u1_amd64.deb ...
Unpacking g++-12 (12.2.0-14+deb12u1) ...
Selecting previously unselected package g++.
Preparing to unpack .../33-g++_4%3a12.2.0-3_amd64.deb ...
Unpacking g++ (4:12.2.0-3) ...
Selecting previously unselected package make.
Preparing to unpack .../34-make_4.3-4.1_amd64.deb ...
Unpacking make (4.3-4.1) ...
Selecting previously unselected package libdpkg-perl.
Preparing to unpack .../35-libdpkg-perl_1.21.22_all.deb ...
Unpacking libdpkg-perl (1.21.22) ...
Selecting previously unselected package patch.
Preparing to unpack .../36-patch_2.7.6-7_amd64.deb ...
Unpacking patch (2.7.6-7) ...
Selecting previously unselected package dpkg-dev.
Preparing to unpack .../37-dpkg-dev_1.21.22_all.deb ...
Unpacking dpkg-dev (1.21.22) ...
Selecting previously unselected package build-essential.
Preparing to unpack .../38-build-essential_12.9_amd64.deb ...
Unpacking build-essential (12.9) ...
Selecting previously unselected package libfakeroot:amd64.
Preparing to unpack .../39-libfakeroot_1.31-1.2_amd64.deb ...
Unpacking libfakeroot:amd64 (1.31-1.2) ...
Selecting previously unselected package fakeroot.
Preparing to unpack .../40-fakeroot_1.31-1.2_amd64.deb ...
Unpacking fakeroot (1.31-1.2) ...
Selecting previously unselected package fonts-dejavu-core.
Preparing to unpack .../41-fonts-dejavu-core_2.37-6_all.deb ...
Unpacking fonts-dejavu-core (2.37-6) ...
Selecting previously unselected package fontconfig-config.
Preparing to unpack .../42-fontconfig-config_2.14.1-4_amd64.deb ...
Unpacking fontconfig-config (2.14.1-4) ...
Selecting previously unselected package libabsl20220623:amd64.
Preparing to unpack .../43-libabsl20220623_20220623.1-1+deb12u2_amd64.deb ...
Unpacking libabsl20220623:amd64 (20220623.1-1+deb12u2) ...
Selecting previously unselected package libalgorithm-diff-perl.
Preparing to unpack .../44-libalgorithm-diff-perl_1.201-1_all.deb ...
Unpacking libalgorithm-diff-perl (1.201-1) ...
Selecting previously unselected package libalgorithm-diff-xs-perl:amd64.
Preparing to unpack .../45-libalgorithm-diff-xs-perl_0.04-8+b1_amd64.deb ...
Unpacking libalgorithm-diff-xs-perl:amd64 (0.04-8+b1) ...
Selecting previously unselected package libalgorithm-merge-perl.
Preparing to unpack .../46-libalgorithm-merge-perl_0.08-5_all.deb ...
Unpacking libalgorithm-merge-perl (0.08-5) ...
Selecting previously unselected package libaom3:amd64.
Preparing to unpack .../47-libaom3_3.6.0-1+deb12u2_amd64.deb ...
Unpacking libaom3:amd64 (3.6.0-1+deb12u2) ...
Selecting previously unselected package libdav1d6:amd64.
Preparing to unpack .../48-libdav1d6_1.0.0-2+deb12u1_amd64.deb ...
Unpacking libdav1d6:amd64 (1.0.0-2+deb12u1) ...
Selecting previously unselected package libgav1-1:amd64.
Preparing to unpack .../49-libgav1-1_0.18.0-1+b1_amd64.deb ...
Unpacking libgav1-1:amd64 (0.18.0-1+b1) ...
Selecting previously unselected package librav1e0:amd64.
Preparing to unpack .../50-librav1e0_0.5.1-6_amd64.deb ...
Unpacking librav1e0:amd64 (0.5.1-6) ...
Selecting previously unselected package libsvtav1enc1:amd64.
Preparing to unpack .../51-libsvtav1enc1_1.4.1+dfsg-1_amd64.deb ...
Unpacking libsvtav1enc1:amd64 (1.4.1+dfsg-1) ...
Selecting previously unselected package libjpeg62-turbo:amd64.
Preparing to unpack .../52-libjpeg62-turbo_1%3a2.1.5-2_amd64.deb ...
Unpacking libjpeg62-turbo:amd64 (1:2.1.5-2) ...
Selecting previously unselected package libyuv0:amd64.
Preparing to unpack .../53-libyuv0_0.0~git20230123.b2528b0-1_amd64.deb ...
Unpacking libyuv0:amd64 (0.0~git20230123.b2528b0-1) ...
Selecting previously unselected package libavif15:amd64.
Preparing to unpack .../54-libavif15_0.11.1-1+deb12u1_amd64.deb ...
Unpacking libavif15:amd64 (0.11.1-1+deb12u1) ...
Selecting previously unselected package libpng16-16:amd64.
Preparing to unpack .../55-libpng16-16_1.6.39-2_amd64.deb ...
Unpacking libpng16-16:amd64 (1.6.39-2) ...
Selecting previously unselected package libfreetype6:amd64.
Preparing to unpack .../56-libfreetype6_2.12.1+dfsg-5+deb12u4_amd64.deb ...
Unpacking libfreetype6:amd64 (2.12.1+dfsg-5+deb12u4) ...
Selecting previously unselected package libfontconfig1:amd64.
Preparing to unpack .../57-libfontconfig1_2.14.1-4_amd64.deb ...
Unpacking libfontconfig1:amd64 (2.14.1-4) ...
Selecting previously unselected package libde265-0:amd64.
Preparing to unpack .../58-libde265-0_1.0.11-1+deb12u2_amd64.deb ...
Unpacking libde265-0:amd64 (1.0.11-1+deb12u2) ...
Selecting previously unselected package libnuma1:amd64.
Preparing to unpack .../59-libnuma1_2.0.16-1_amd64.deb ...
Unpacking libnuma1:amd64 (2.0.16-1) ...
Selecting previously unselected package libx265-199:amd64.
Preparing to unpack .../60-libx265-199_3.5-2+b1_amd64.deb ...
Unpacking libx265-199:amd64 (3.5-2+b1) ...
Selecting previously unselected package libheif1:amd64.
Preparing to unpack .../61-libheif1_1.15.1-1+deb12u1_amd64.deb ...
Unpacking libheif1:amd64 (1.15.1-1+deb12u1) ...
Selecting previously unselected package libdeflate0:amd64.
Preparing to unpack .../62-libdeflate0_1.14-1_amd64.deb ...
Unpacking libdeflate0:amd64 (1.14-1) ...
Selecting previously unselected package libjbig0:amd64.
Preparing to unpack .../63-libjbig0_2.1-6.1_amd64.deb ...
Unpacking libjbig0:amd64 (2.1-6.1) ...
Selecting previously unselected package liblerc4:amd64.
Preparing to unpack .../64-liblerc4_4.0.0+ds-2_amd64.deb ...
Unpacking liblerc4:amd64 (4.0.0+ds-2) ...
Selecting previously unselected package libwebp7:amd64.
Preparing to unpack .../65-libwebp7_1.2.4-0.2+deb12u1_amd64.deb ...
Unpacking libwebp7:amd64 (1.2.4-0.2+deb12u1) ...
Selecting previously unselected package libtiff6:amd64.
Preparing to unpack .../66-libtiff6_4.5.0-6+deb12u2_amd64.deb ...
Unpacking libtiff6:amd64 (4.5.0-6+deb12u2) ...
Selecting previously unselected package libxau6:amd64.
Preparing to unpack .../67-libxau6_1%3a1.0.9-1_amd64.deb ...
Unpacking libxau6:amd64 (1:1.0.9-1) ...
Selecting previously unselected package libxdmcp6:amd64.
Preparing to unpack .../68-libxdmcp6_1%3a1.1.2-3_amd64.deb ...
Unpacking libxdmcp6:amd64 (1:1.1.2-3) ...
Selecting previously unselected package libxcb1:amd64.
Preparing to unpack .../69-libxcb1_1.15-1_amd64.deb ...
Unpacking libxcb1:amd64 (1.15-1) ...
Selecting previously unselected package libx11-data.
Preparing to unpack .../70-libx11-data_2%3a1.8.4-2+deb12u2_all.deb ...
Unpacking libx11-data (2:1.8.4-2+deb12u2) ...
Selecting previously unselected package libx11-6:amd64.
Preparing to unpack .../71-libx11-6_2%3a1.8.4-2+deb12u2_amd64.deb ...
Unpacking libx11-6:amd64 (2:1.8.4-2+deb12u2) ...
Selecting previously unselected package libxpm4:amd64.
Preparing to unpack .../72-libxpm4_1%3a3.5.12-1.1+deb12u1_amd64.deb ...
Unpacking libxpm4:amd64 (1:3.5.12-1.1+deb12u1) ...
Selecting previously unselected package libgd3:amd64.
Preparing to unpack .../73-libgd3_2.3.3-9_amd64.deb ...
Unpacking libgd3:amd64 (2.3.3-9) ...
Selecting previously unselected package libc-devtools.
Preparing to unpack .../74-libc-devtools_2.36-9+deb12u13_amd64.deb ...
Unpacking libc-devtools (2.36-9+deb12u13) ...
Selecting previously unselected package libfile-fcntllock-perl.
Preparing to unpack .../75-libfile-fcntllock-perl_0.22-4+b1_amd64.deb ...
Unpacking libfile-fcntllock-perl (0.22-4+b1) ...
Selecting previously unselected package manpages-dev.
Preparing to unpack .../76-manpages-dev_6.03-2_all.deb ...
Unpacking manpages-dev (6.03-2) ...
Setting up libaom3:amd64 (3.6.0-1+deb12u2) ...
Setting up manpages-dev (6.03-2) ...
Setting up libabsl20220623:amd64 (20220623.1-1+deb12u2) ...
Setting up libxau6:amd64 (1:1.0.9-1) ...
Setting up libxdmcp6:amd64 (1:1.1.2-3) ...
Setting up libxcb1:amd64 (1.15-1) ...
Setting up liblerc4:amd64 (4.0.0+ds-2) ...
Setting up libfile-fcntllock-perl (0.22-4+b1) ...
Setting up libalgorithm-diff-perl (1.201-1) ...
Setting up binutils-common:amd64 (2.40-2) ...
Setting up libdeflate0:amd64 (1.14-1) ...
Setting up linux-libc-dev:amd64 (6.1.153-1) ...
Setting up libctf-nobfd0:amd64 (2.40-2) ...
Setting up libsvtav1enc1:amd64 (1.4.1+dfsg-1) ...
Setting up libgomp1:amd64 (12.2.0-14+deb12u1) ...
Setting up libjbig0:amd64 (2.1-6.1) ...
Setting up librav1e0:amd64 (0.5.1-6) ...
Setting up libfakeroot:amd64 (1.31-1.2) ...
Setting up fakeroot (1.31-1.2) ...
update-alternatives: using /usr/bin/fakeroot-sysv to provide /usr/bin/fakeroot (fakeroot) in auto mode
Setting up libtirpc-dev:amd64 (1.3.3+ds-1) ...
Setting up rpcsvc-proto (1.4.3-1) ...
Setting up libjpeg62-turbo:amd64 (1:2.1.5-2) ...
Setting up libx11-data (2:1.8.4-2+deb12u2) ...
Setting up make (4.3-4.1) ...
Setting up libmpfr6:amd64 (4.2.0-1) ...
Setting up libquadmath0:amd64 (12.2.0-14+deb12u1) ...
Setting up libpng16-16:amd64 (1.6.39-2) ...
Setting up libmpc3:amd64 (1.3.1-1) ...
Setting up libatomic1:amd64 (12.2.0-14+deb12u1) ...
Setting up patch (2.7.6-7) ...
Setting up fonts-dejavu-core (2.37-6) ...
Setting up libgav1-1:amd64 (0.18.0-1+b1) ...
Setting up libdav1d6:amd64 (1.0.0-2+deb12u1) ...
Setting up libdpkg-perl (1.21.22) ...
Setting up libwebp7:amd64 (1.2.4-0.2+deb12u1) ...
Setting up libubsan1:amd64 (12.2.0-14+deb12u1) ...
Setting up libnuma1:amd64 (2.0.16-1) ...
Setting up libnsl-dev:amd64 (1.3.0-2) ...
Setting up libcrypt-dev:amd64 (1:4.4.33-2) ...
Setting up libtiff6:amd64 (4.5.0-6+deb12u2) ...
Setting up libasan8:amd64 (12.2.0-14+deb12u1) ...
Setting up libx11-6:amd64 (2:1.8.4-2+deb12u2) ...
Setting up libtsan2:amd64 (12.2.0-14+deb12u1) ...
Setting up libbinutils:amd64 (2.40-2) ...
Setting up libisl23:amd64 (0.25-1.1) ...
Setting up libde265-0:amd64 (1.0.11-1+deb12u2) ...
Setting up libc-dev-bin (2.36-9+deb12u13) ...
Setting up libyuv0:amd64 (0.0~git20230123.b2528b0-1) ...
Setting up libalgorithm-diff-xs-perl:amd64 (0.04-8+b1) ...
Setting up libcc1-0:amd64 (12.2.0-14+deb12u1) ...
Setting up liblsan0:amd64 (12.2.0-14+deb12u1) ...
Setting up libitm1:amd64 (12.2.0-14+deb12u1) ...
Setting up libalgorithm-merge-perl (0.08-5) ...
Setting up libctf0:amd64 (2.40-2) ...
Setting up cpp-12 (12.2.0-14+deb12u1) ...
Setting up libxpm4:amd64 (1:3.5.12-1.1+deb12u1) ...
Setting up libavif15:amd64 (0.11.1-1+deb12u1) ...
Setting up fontconfig-config (2.14.1-4) ...
Setting up libgprofng0:amd64 (2.40-2) ...
Setting up libfreetype6:amd64 (2.12.1+dfsg-5+deb12u4) ...
Setting up libgcc-12-dev:amd64 (12.2.0-14+deb12u1) ...
Setting up libx265-199:amd64 (3.5-2+b1) ...
Setting up cpp (4:12.2.0-3) ...
Setting up libc6-dev:amd64 (2.36-9+deb12u13) ...
Setting up libfontconfig1:amd64 (2.14.1-4) ...
Setting up binutils-x86-64-linux-gnu (2.40-2) ...
Setting up libstdc++-12-dev:amd64 (12.2.0-14+deb12u1) ...
Setting up libheif1:amd64 (1.15.1-1+deb12u1) ...
Setting up binutils (2.40-2) ...
Setting up dpkg-dev (1.21.22) ...
Setting up gcc-12 (12.2.0-14+deb12u1) ...
Setting up libgd3:amd64 (2.3.3-9) ...
Setting up libc-devtools (2.36-9+deb12u13) ...
Setting up g++-12 (12.2.0-14+deb12u1) ...
Setting up gcc (4:12.2.0-3) ...
Setting up g++ (4:12.2.0-3) ...
update-alternatives: using /usr/bin/g++ to provide /usr/bin/c++ (c++) in auto mode
Setting up build-essential (12.9) ...
Processing triggers for man-db (2.11.2-2) ...
Processing triggers for libc-bin (2.36-9+deb12u13) ...
  ✔️   Installed Dependencies
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'odoo' instead of '/opt/odoo.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 odoo : Depends: python3-lxml-html-clean but it is not installable
        Recommends: postgresql but it is not going to be installed
        Recommends: python3-ldap but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

[ERROR] in line 31: exit code 0: while executing command $STD apt install -y /opt/odoo.deb

[ERROR] in line 1345: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://git.community-scripts.org/community-scripts/ProxmoxVE/raw/branch/main/install/${var_install}.sh)"


### 🖼️ Additional context (optional).

_No response_

--- a/ct/odoo.sh
+++ b/ct/odoo.sh
@@ -28,6 +28,13 @@ function update_script() {
     msg_error "No ${APP} Installation Found!"
     exit 1
   fi
+  if ! [[ $(dpkg -s python3-lxml-html-clean 2>/dev/null) ]]; then
+    $STD apt-get install python-lxml
+    curl -fsSL "http://archive.ubuntu.com/ubuntu/pool/universe/l/lxml-html-clean/python3-lxml-html-clean_0.1.1-1_all.deb" -o /opt/python3-lxml-html-clean.deb
+    $STD dpkg -i /opt/python3-lxml-html-clean.deb
+    rm -f /opt/python3-lxml-html-clean.deb
+  fi
+
   RELEASE=$(curl -fsSL https://nightly.odoo.com/ | grep -oE 'href="[0-9]+\.[0-9]+/nightly"' | head -n1 | cut -d'"' -f2 | cut -d/ -f1)
   LATEST_VERSION=$(curl -fsSL "https://nightly.odoo.com/${RELEASE}/nightly/deb/" |
     grep -oP "odoo_${RELEASE}\.\d+_all\.deb" |

--- a/install/odoo-install.sh
+++ b/install/odoo-install.sh
@@ -16,8 +16,13 @@ update_os
 msg_info "Installing Dependencies"
 $STD apt-get install -y \
   build-essential \
-  make
+  make \
+  python3-lxml
+curl -fsSL "http://archive.ubuntu.com/ubuntu/pool/universe/l/lxml-html-clean/python3-lxml-html-clean_0.1.1-1_all.deb" -o /opt/python3-lxml-html-clean.deb
+$STD dpkg -i /opt/python3-lxml-html-clean.deb
 msg_ok "Installed Dependencies"
+
+PG_VERSION="18" setup_postgresql
 
 RELEASE=$(curl -fsSL https://nightly.odoo.com/ | grep -oE 'href="[0-9]+\.[0-9]+/nightly"' | head -n1 | cut -d'"' -f2 | cut -d/ -f1)
 LATEST_VERSION=$(curl -fsSL "https://nightly.odoo.com/${RELEASE}/nightly/deb/" |
@@ -69,6 +74,7 @@ customize
 
 msg_info "Cleaning up"
 rm -f /opt/odoo.deb
+rm -f /opt/python3-lxml-html-clean.deb
 $STD apt-get autoremove
 $STD apt-get autoclean
 msg_ok "Cleaned"

--- a/install/odoo-install.sh
+++ b/install/odoo-install.sh
@@ -14,10 +14,7 @@ network_check
 update_os
 
 msg_info "Installing Dependencies"
-$STD apt-get install -y \
-  build-essential \
-  make \
-  python3-lxml
+$STD apt-get install -y python3-lxml
 curl -fsSL "http://archive.ubuntu.com/ubuntu/pool/universe/l/lxml-html-clean/python3-lxml-html-clean_0.1.1-1_all.deb" -o /opt/python3-lxml-html-clean.deb
 $STD dpkg -i /opt/python3-lxml-html-clean.deb
 msg_ok "Installed Dependencies"

